kratos/python_scripts/analysis_stage.py

```diff
@@ -128,7 +128,7 @@ def InitializeSolutionStep(self):
         self.ChangeMaterialProperties() #this is normally empty
         self._GetSolver().InitializeSolutionStep()
 
-        self.PrintAnalysisStageProgressInformation()
+        #self.PrintAnalysisStageProgressInformation()
```

Comment:
please revert this

ideally you should not commit changes to the core files, this will show up in the history and makes it e.g. more complicated for tracing problems

Reply:
> Hi @farrufat-cimne , please take a look at this file, too.
> 
> https://github.com/KratosMultiphysics/Kratos/blob/e9098127ca07151a0fcfc521029973d549234f4f/applications/DEMApplication/python_scripts/DEM_material_test_script.py#L466
> 
> (and following lines!)
> 
> https://github.com/KratosMultiphysics/Kratos/blob/e9098127ca07151a0fcfc521029973d549234f4f/applications/DEMApplication/python_scripts/DEM_material_test_script.py#L390
> 
> (and following lines!)
> 
> This is a dangerous PR and we need to make sure that we do not forget any place where we are using variables related to stress.

For aesthetics, it may not be pleasing to look at it. 
As far as code functionality is concerned I see no issue in there since they do not depend on the mdpa values.
I will further review it anyway, just in case